Resources/views/MediaAdmin/edit.html.twig

```diff
@@ -50,8 +67,7 @@ file that was distributed with this source code.
                             <table class="table">
                                 <tr>
                                     <th>{{ 'label.size'|trans({}, 'SonataMediaBundle') }}</th>
-                                    <td>{{ object.width|number_format_decimal }}px x {{ object.height|number_format_decimal }}px
-                                        {% if object.size > 0 %}({{ object.size|number_format_decimal }}o){% endif %}</td>
+                                    <td>{{ object.width }} x {{ object.height }} {% if object.size > 0 %}({{ macros.bytesToSize(object.size) }}){% endif %}</td>
```

Comment:
I was thinking on using number_format(0), just to avoid decimals (because decimals are problematic with . and , depending on country.

For the other part, for me it is hard to read: 350px x 500px. Feel it reads better with just 350 x 500. If we are already talking about size, we know thats pixels.